sources/packages/forms/src/form-definitions/educationprogramoffering.json

```diff
@@ -2035,28 +2106,72 @@
                                 "minMode": "day",
                                 "maxMode": "year",
                                 "yearRows": 4,
-                                "yearColumns": 5,
-                                "minDate": null,
-                                "maxDate": null
+                                "yearColumns": 5
                               },
                               "enableMaxDateInput": false,
                               "enableTime": false,
+                              "timePicker": {

```

Comment:
why is the timepicker options present when enableTIme is false. I saw some existing form definitions and it did not have timePicker property in it?